ratelimit/ratelimiting.go

```diff
@@ -32,9 +33,14 @@ import (
 	"time"
 )
 
+// nolint: structcheck
 type Bucket struct {
 	sync.Mutex
+	// avoid `limit` is unused (unused)
+	// nolint: unused
 	limit     int
+	// avoid `fillLevel` is unused (unused)
+	// nolint: unused
 	fillLevel int
```

Comment:
Again; if these fields are *actually* unused, let's get rid of them for good. 🗑️

Reply:
:+1: